Session-6/task3/main.go

```diff
@@ -7,4 +7,5 @@ func main() {
 	fmt.Printf("len is = %d cap is = %d\n", len(an), cap(an))
 	an = []int{1, 2, 3, 4, 5}
```

Comment:
Burda yeni slice formalaşır. Hərçənd mövcud slice-da modifikasiya edib, dəyişən cap və len dəyərlərini görmək daha maraqlıdır
```suggestion
	an = append(an, 1, 2, 3, 4, 5)
```